src/app/commands/cmd_new_brush.cpp

```diff
@@ -64,6 +64,10 @@ bool NewBrushCommand::onEnabled(Context* context)
 
 void NewBrushCommand::onExecute(Context* context)
 {
+  ASSERT(current_editor);
+  if (!current_editor)
+    return;
+
```

Comment:
Is this really needed? The onEnabled() condition should filter this case. (Anyway you may have added this for some special case.)